examples/md-flexible/measurePerf.sh

```diff
@@ -81,73 +81,87 @@ do
             # loop for different verlet rebuild frequencies and skins
             for iVL in `seq 0 $(( ${#VLRebuild[@]} - 1 ))` ;
             do
-                configPrinted=false
-
-                # since this loop only has one iteration for non verlet container only print for verlet
-                if [[ ${container} =~ 'Verlet' ]];
-                then
-                    separate "VLRebuild: ${VLRebuild[$iVL]} VLSkin: ${VLSkin[$iVL]}"
-                    filename="runtimes_${container}_${dataLayout}_N3${newton3Opt}_${VLRebuild[$iVL]}_${VLSkin[$iVL]}.csv"
-                else
-                    filename="runtimes_${container}_${dataLayout}_N3${newton3Opt}.csv"
-                fi
-
-                # workaround because there is no traversal for Verlet clusters with newton 3 yet.
-                if [[ ${container} =~ 'VerletCluster' && ${newton3Opt} =~ 'on' ]];
-                then
-                    continue
-                fi
-
-                # iterate over molecules with the correct repetition
-                for i in `seq 0 $(( ${#Mols[@]} - 1 ))` ;
+                for cellSize in 1.0  1.25  1.5  1.75  2.0  2.5  3.0;
```

Comment:
Testing multiple cell sizes does not make sense for all containers. Restrict this only to LinkedCells for now.
Also, I think 7 values are a bit excessive :D Choose 3 which you think are interesting.